src/dawn_native/ShaderModule.cpp

```diff
@@ -114,12 +115,11 @@ namespace dawn_native {
         auto ExtractResourcesBinding = [this](const std::vector<spirv_cross::Resource>& resources,
                                               const spirv_cross::Compiler& compiler,
                                               dawn::BindingType bindingType) {
-            constexpr uint64_t requiredBindingDecorationMask =
-                (1ull << spv::DecorationBinding) | (1ull << spv::DecorationDescriptorSet);
-
             for (const auto& resource : resources) {
-                ASSERT((compiler.get_decoration_mask(resource.id) &
-                        requiredBindingDecorationMask) == requiredBindingDecorationMask);
+                ASSERT(compiler.get_decoration_bitset(resource.id).get(spv::DecorationBinding));
```

Comment:
I assume these changes are necessary to roll spirv-cross, not unrelated?

Reply:
Yep, the functions we were using before were marked as deprecated and caused warnings hence errors.